sql/log_event_server.cc

```diff
@@ -2457,7 +2457,7 @@ static void check_and_remove_stale_alter(Relay_log_info *rli)
   {
     DBUG_ASSERT(info->state == start_alter_state::REGISTERED);
 
-    sql_print_warning("ALTER query started at %u-%u-%llu could not "
+    sql_print_warning("ALTER query started at %llu-%u-%u could not "
```

Comment:
I would use `PRIu64` over `%llu` for the `uint64` here, but somehow it doesn’t work, at least on my (WSL) Ubuntu.

Reply:
I had it extracted as a part of #3518.

I did not file separate bug fix tickets for any PR listed on the top post, as I did not know all of our conventions.
Mentor @LinuxJedi[^m] merged #3541 but I’m not aware of any separate JIRA tickets.

[^m]: Reminder: This project was a GSoC contribution I submitted before the organization accepted me as a member. Notice how the PRs are sourcing from my personal fork. (:

Reply:
> Please file a separate JIRA for each version that is getting backports.

What about #3541 which is already merged?

Reply:
> I had it extracted as a part of https://github.com/MariaDB/server/pull/3518.

I saw :) It is great that you had already created backports for these problems.

> What about https://github.com/MariaDB/server/pull/3541 which is already merged?

Please create one. I'd say anything that a user may face should be searchable in JIRA (though it doesn't necessarily have to be a JIRA of its own, it can be part of a bigger JIRA with a bunch of log message fixes that all have the same fix version). Though as it has already been merged (without a git commit mentioning the reference JIRA), I'd say to paste the commit hash in the description of the JIRA.

Reply:
> `%u-%u-%llu` should be filled in by: `info->domain_id`, `rli->mi->master_id`, and `info->sa_seq_no`.

It’s a GTID! I wouldn’t know back then, but now I know what a GTID is. 😄
I’ve filled it in and pushed to #3518.

Reply:
> 1. Please file a separate JIRA for each version that is getting backports.
> 2. For each bugged error message that you fixed (nice job btw!), I'd say either mention them in the backport-JIRA you file, or create separate JIRAs for them (and mention them in the commit message). […]
> ⸺ [review comment](#pullrequestreview-2421203312)

I have created [an MDEV-21978 subtask](https://jira.mariadb.org/browse/MDEV-21978#view-subtasks-label) for each backport by version, such as:
* > I'd say we should file a JIRA and fix this in the 10.11 backport.
  
  I have included this in [MDEV-35431](https://jira.mariadb.org/browse/MDEV-35431).
* > > What about #3541 which is already merged?
  > 
  > […] Though as it has already been merged (without a git commit mentioning the reference JIRA), I'd say to paste the commit hash in the description of the JIRA.
  
  I have filed and closed [MDEV-35433](https://jira.mariadb.org/browse/MDEV-35433). The PR and commit links are in the comments.

I have also added JIRA links to each PR. JIRA will pick them up. (Though [MDEV-35429](https://jira.mariadb.org/browse/MDEV-35429) took a day to pick up #3485.)